meta/labs/meetings/15-09-03_who_writes_the_initial_playbook.md

```diff
@@ -0,0 +1,69 @@
+# Who Writes The Initial Draft of The Playbook?
+
+--------------------------------------------------------------------------------
+_Date:_
```

Comment:
As mentioned in https://github.com/hackedu/meta/blob/2360c50372eb331cc46c67e6faf5bcdb7d4655d0/markdown_style_guide.md#horizontal-rules, horizontal rules must be followed by a newline.

I'd push for removing this horizontal rule entirely, per the template in https://github.com/hackedu/hackedu/tree/master/meta/labs#meetings.